### PR TITLE
[CSDP] add blegat patch

### PR DESCRIPTION
**Pull request by amontoison**

### CODE REVIEW

**Comment by amontoison** [comment]
@blegat can you have a look? I have some errors with your patch during the compilation.

File: C/Coin-OR/CSDP/bundled/patches/blegat.patch [hunk 1]
```diff
@@ -0,0 +1,1731 @@
...
+   struct constraintmatrix fill;
+-  struct paramstruc params;
+   struct blockmatrix work1;
+   struct blockmatrix work2;
+   struct blockmatrix work3;
+@@ -53,7 +76,6 @@ int easy_sdp(n,k,C,a,constraints,constan
+   double *dy1;
+   double *rhs;
+   double *besty;
+-  int printlevel;
```
Comment (blegat) [inline C/Coin-OR/CSDP/bundled/patches/blegat.patch:181]: It seems there is an issue with this patch. This removal is not applied and it is causing the compilation failure. I tried locally to apply this patch and it is also not removed.

Comment (blegat) [reply C/Coin-OR/CSDP/bundled/patches/blegat.patch:181]: I made a PR in https://github.com/amontoison/Yggdrasil/pull/1 to fix this

Comment (amontoison) [reply C/Coin-OR/CSDP/bundled/patches/blegat.patch:181]: Thanks blegat! I must have made a mistake when I generated the patch. All checks have passed :rocket:

**Pull request merged**